core_impl/src/main/java/com/google/instrumentation/trace/SpanImpl.java

```diff
@@ -54,13 +55,17 @@
   // The start time of the span. Set when the span is created iff the RECORD_EVENTS options is
   // set, otherwise 0.
   private final long startNanoTime;
-  // List of the recorded annotations.
+  // Set of recorded attributes. Eviction is based on the insertion order (note that insertion
+  // order is not affected if a key is re-inserted into the map.)
```

Comment:
I know but that parameter also change the access order based when we do a get, which should not be the case, but fortunately the only getters are iteration over the whole map which should not change the access order.